app/main/config.py

```diff
@@ -42,7 +42,7 @@ class Config:
     VIDEO_MODEL_L1_SCORE = 0.7
   AUDIO_MODEL = os.getenv('AUDIO_MODEL', 'audio-model')
   IMAGE_MODEL = os.getenv('IMAGE_MODEL', default='phash')
-
+  OPENAI_API_KEY = os.getenv('OPENAI_API_KEY', default=None)
```

Comment:
I think it would be helpful to add a comment with the SSM key for the secret so that it is clear what needs to be added to the environment if it is missing

Reply:
Scott have elaborated it to me. And we added 
```# For local development, you can populate this key from SSM.
# The key is /local/alegre/OPENAI_API_KEY```
to the env_file.example